libctru/source/services/nfc.c

```diff
@@ -15,12 +15,14 @@ static NFC_OpType nfc_optype = NFC_OpType_NFCTag;
 static Result NFC_Initialize(NFC_OpType type);
 static Result NFC_Shutdown(NFC_OpType type);
 
+static Result NFC_StartCommsAdapter();
```

Comment:
Should be `(void)` just like the others - in C, `()` doesn't indicate an empty parameter list.